QualifiedTeachersApi/src/QualifiedTeachersApi/Properties/StringResources.resx

```diff
@@ -219,4 +219,7 @@
   <data name="Errors.10023.Title" xml:space="preserve">
     <value>User with specified UserId not found</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="Errors.10026.Title" xml:space="preserve">
+    <value>Teacher cannot change QTS/EYTS date</value>
```

Comment:
This message doesn't make sense to me. The teacher isn't trying to change the date...